cl/users/views.py

```diff
@@ -289,13 +293,24 @@ def view_settings(request: AuthenticatedHttpRequest) -> HttpResponse:
         user_form.save()
 
         return HttpResponseRedirect(reverse("view_settings"))
+
+    email_broken = broken_email_address(user)
+    if email_broken["EMAIL_BROKEN_BANNER"]["show"]:
+        email_error_message = determine_email_error_message(
+            email_broken["EMAIL_BROKEN_BANNER"]["type"],
+            email_broken["EMAIL_BROKEN_BANNER"]["sub_type"],
+        )
+    else:
+        email_error_message = ""
+
     return render(
         request,
         "profile/settings.html",
         {
             "profile_form": profile_form,
             "user_form": user_form,
             "private": True,
+            "email_error_message": email_error_message,
```

Comment:
Should consider removing this and passing the message in via Django msg feature.

Reply:
Removed, we now using the Django message feature instead.